### ISSUE TITLE
Style button in editor inoperative on Android

### ISSUE THREAD

**Issue by alerque** [bibledit/bibledit#107]
Copied from comments on #101:

> On Android 6.0.1 the Bible text editor fails to operate the "Style" button, with the result that styles cannot be called or seen.

**Comment by alerque** [comment]
Which editor layout are we talking about here? Basic mode or advanced mode?

**Comment by teusbenschop** [comment]
It happens in the advanced mode. In basic mode there is no Styles button by
design.

**Comment by alerque** [comment]
> fails to operate

Is the button not showing up at all or does it do nothing when pressed?

**Comment by teusbenschop** [comment]
The button shows up but it does nothing when pressed.

**Comment by alerque** [comment]
This should be closed as a duplicate of #106. The first thing the style button handler does is check if the area is editable and the function does nothing if it is not. If we fix the editable flag problem this should resolve itself.

From [L628 of `lib/edit/index.js`](https://github.com/teusbenschop/bibledit/blob/master/lib/edit/index.js#L628):

``` js
function editorStylesButtonHandler ()
{
  if (!editorWriteAccess) return;
 ...
}
```

**Comment by teusbenschop** [comment]
You are right. Closing...
